components/claim/create-reward-claim.tsx

```diff
@@ -174,12 +227,39 @@ const CreateRewardClaim = () => {
         data?.ipfsHash as string
       );
     } catch (error) {
+      showCustomErrorToast("Error creating new claimable batch", width);
       console.error("Error creating new claimable batch:", error);
     } finally {
       setLoading(false);
     }
   }, [createNewClaimableBatch, claimInput, claimDeadlineAsString]);
 
+  useEffect(() => {
+    if (approveError) {
+      showCustomErrorToast("Error approving claimable funds", width);
+      setLoading(false);
+      return;
+    }
+
+    if (isConfirmed) {
+      showSuccessToast("Successfully created new claimable batch.", width);
+      setLoading(false);
+      return;
+    }
+
+    if (error) {
+      showCustomErrorToast(error.message, width);
+      setLoading(false);
+      return;
+    }
+
+    if (approvedSuccess) {
+      showSuccessToast("Successfully approved claimable funds.", width);
+      setLoading(false);
+      return;
+    }
+  }, [approveError, approvedSuccess, isConfirmed, error]);
+
```

Comment:
we can also consider calling reset() on all tx once an input changes, that way it will prevent the same error messages from popping up if it happens to refetch

Reply:
here is the documentation:
https://wagmi.sh/react/api/hooks/useWriteContracts#reset

Reply:
I think I did this..? Please lmk :)